main.go

```diff
@@ -34,6 +36,8 @@ func init() {
 	flag.IntVar(&flagRedisDB, "redis_db", 0, "redis database number")
 	flag.StringVar(&flagRedisPassword, "redis_password", "", "password to use when connecting to redis server")
 	flag.StringVar(&flagRedisTLS, "redis_tls", "", "server name for TLS validation used when connecting to redis server")
```

Comment:
Can we also update the other flags to use "-" (dash) instead of "_" (underscore) for consistency?

Reply:
Done. I left them in place mainly for backwards compatibility. If you're okay with breaking them I would suggest just removing them in favor of redis-url since there would be two ways to specify them and it could be confusing.

I've pushed a commit that would do that to another branch. You can see what it would look like here:
https://github.com/hibiken/asynqmon/commit/f2a8b78ec3ea984acec9d4d05237546782f7ed45

Reply:
Ah good point. Didn't think of command line flags as an API, but you're right. I will follow up to this PR and think if we should remove the flags or update them to be in a more consistent format.

`redis-cli` seem to support both `uri` option and `hostname`, `port`, `password`, etc. So I may keep both options.